R/LongitudinalGSF.R

```diff
@@ -36,14 +36,14 @@ NULL
 #' @export
 LongitudinalGSF <- function(
     mu_bsld = prior_lognormal(log(55), 5, init = 55),
-    mu_ks = prior_lognormal(0, 0.5, init = 0),
-    mu_kg = prior_lognormal(-0.36, 1, init = -0.36),
+    mu_ks = prior_lognormal(0, 0.5, init = 0.01),
+    mu_kg = prior_lognormal(-0.36, 1, init = 0.01),
```

Comment:
These initial values put them in the boundaries of the stan code, but I wonder if we should use the same construction as for `mu_bsld` by specifying the mean as a `log()` (or `init` as `exp()`) for clarity.

Reply:
Sounds good, thanks @gravesti